lib/cartopy/crs.py

```diff
@@ -1271,8 +1271,10 @@ def threshold(self):
 
 
 class Gnomonic(Projection):
-    def __init__(self, central_latitude=0.0, globe=None):
-        proj4_params = [('proj', 'gnom'), ('lat_0', central_latitude)]
+    def __init__(self, central_latitude=0.0, 
```

Comment:
This line has some whitespace at the end, the last character on the line should be the `,`.